Casks/zulu8.rb

```diff
@@ -1,29 +1,36 @@
 cask "zulu8" do
-  version "8.0.292,8.54.0.21-ca"
-
   if Hardware::CPU.intel?
-    sha256 "a60a31b33a8386f9b446cad2615e33d0190eb0f18cdf8d6812b74fd6e0f4f716"
-
+    version "8.0.302,8.56.0.21-ca"
+    sha256 "4e667ac2b946d8890a0d9ffe2168e1ad1ccb3f82c3683ccd3f10ce5349c874a7"
     url "https://cdn.azul.com/zulu/bin/zulu#{version.after_comma}-jdk#{version.before_comma}-macosx_x64.dmg",
         referer: "https://www.azul.com/downloads/zulu/zulu-mac/"
+    
+    livecheck do
+      url "https://api.azul.com/zulu/download/community/v1.0/bundles/latest/?jdk_version=#{version.major}&ext=dmg&os=macos&arch=x86"
+      strategy :page_match do |page|
+        match = page.match(%r{url"\s*:\s*"https:.*?/zulu(\d+(?:\.\d+)*-.*?)-jdk(\d+(?:\.\d+)*)-macos}i)
+        "#{match[2]},#{match[1]}"
+      end
+    end
   else
-    sha256 "78bfc25d73568f94a2f3f46be4ef2f4723250f8a7b3691e68f33528e7517d2d4"
-
+    version "8.0.302,8.56.0.23-ca"
+    sha256 "ee6c3ceaa3ef384c322c50b64feeaf0e7a9b54895300a4935b50a8e5df22e858"
     url "https://cdn.azul.com/zulu/bin/zulu#{version.after_comma}-jdk#{version.before_comma}-macosx_aarch64.dmg",
         referer: "https://www.azul.com/downloads/zulu/zulu-mac/"
-  end
-
-  name "Azul Zulu Java 8 Standard Edition Development Kit"
-  homepage "https://www.azul.com/downloads/zulu/zulu-mac/"
-
-  livecheck do
-    url "https://api.azul.com/zulu/download/community/v1.0/bundles/latest/?jdk_version=#{version.major}&ext=dmg&os=macos"
-    regex(/url":"https:.*zulu(.*)-jdk(.*)-macos/i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| match[1] + "," + match[0] }
+    
+    livecheck do
+      url "https://api.azul.com/zulu/download/community/v1.0/bundles/latest/?jdk_version=#{version.major}&ext=dmg&os=macos&arch=arm"
+      strategy :page_match do |page|
+        match = page.match(%r{url"\s*:\s*"https:.*?/zulu(\d+(?:\.\d+)*-.*?)-jdk(\d+(?:\.\d+)*)-macos}i)
+        "#{match[2]},#{match[1]}"
+      end
     end
   end
 
+  name "Azul Zulu Java 8 Standard Edition Development Kit"
+  desc "OpenJDK distribution from Azul"
+  homepage "https://www.azul.com/downloads/"
```

Comment:
```suggestion
  homepage "https://www.azul.com/products/core/"
```

To match other Zulu versioned casks.

Reply:
I changed to the main homepage.